staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/metrics/metrics.go

```diff
@@ -156,6 +156,16 @@ var (
 		},
 		[]string{"provider_name", "error"},
 	)
+
+	DekCacheFilled = metrics.NewGauge(
```

Comment:
Does this need to be exported?

Reply:
yes, `metrics.DekSourceCacheSize.Reset()` is called in envelope_test.go